tasks/common.js

```diff
@@ -233,15 +233,12 @@ module.exports = function (grunt) {
         return _platform;
     }
     
-    function arch() {
-        if (platform() === "linux") {
-            if (process.arch === "x64") {
-                return 64;
-            } else {
-                return 32;
-            }
-        }
-        
+    function arch(hint) {
+		hint = hint ? hint : process.arch;
+		hint = platform() === 'mac' ? undefined : hint;
```

Comment:
In case of Mac we are returning "" - I'm guessing that is expected.
hint = (platform() !== 'mac' && hint) ? hint : ""; is more concise and precise.

Reply:
Shouldn't we just return with "" in case of mac. Linux and Win will have similar behaviour after that.